tests/integration/helpers.bash

```diff
@@ -132,7 +132,8 @@ function init_cgroup_paths() {
 		CGROUP_SUBSYSTEMS=$(awk '!/^#/ {print $1}' /proc/cgroups)
 		local g base_path
 		for g in ${CGROUP_SUBSYSTEMS}; do
-			base_path=$(awk '$(NF-2) == "cgroup" && $NF ~ /\<'"${g}"'\>/ { print $5; exit }' /proc/self/mountinfo)
+			# This uses gawk-specific feature (\< ... \>).
```

Comment:
How about add some more words? For example:
# This uses gawk-specific feature (\< ... \>), which is not supported  by `awk` without `gawk` installed.

Reply:
@lifubang "gawk-specific" here means the feature is specific to gawk, meaning it is supported by gawk only and not supported by other awk implementation. To my mind, this is enough.

By specifically calling `gawk` here, we ensure we can use gawk-specific feature. We add a comment explaining why we call `gawk and not `awk` and what gawk-specific feature we use.

> not supported by awk without gawk installed

This wording implies that `awk` needs `gawk` or something like this. This is not quite accurate so I would avoid saying that.